internal/envoy/install_test.go

```diff
@@ -267,5 +262,8 @@ func setupInstallTest(t *testing.T) (*installTest, func()) {
 		},
 	}
 	setup.GetEnvoyVersions = NewGetVersions(setup.EnvoyVersionsURL, setup.Platform, setup.Version)
-	return setup, versionsServer.Close
+	t.Cleanup(func() {
```

Comment:
please use `versionsServer.Close` instead of an anonymous function to do the same